CONTRIBUTING.md

```diff
@@ -1,52 +1,160 @@
+
+After you submit your pull request, please assign 3 or less reviewers following our [assigning reviewers best practices](#assigning-reviewers-best-pratices) and assign any [labels](#labels) you think are applicable. Please also verify that all status checks are passing. If for some reason they are failing and you believe it’s unrelated to your change or unsure why, please leave a comment on the pull request explaining this and someone on the gliff.ai team will further investigate.
 
 ## Linting, Formatting and Testing
 
-We are using flake8 and black for linting and auto-formatting our python code and eslint and prettier.io for linting and autoformatting our TypeScript.
-Configuration files can be found in our repos.
+[{{back to navigation}}](#table-of-contents)
 
-We are using pytest for our Python tests and jest with react-testing-library for our TypeScript testing.
-Configuration files can be found in our repos.
+Our teams uses flake8 and black for linting and auto-formatting our Python code alongside pytest for our Python tests, and then eslint and prettier.io for linting and auto-formatting our TypeScript code alongside jest with react-testing-library for our TypeScript testing. Further specifics and configuration files can be found in each of our repositories.
 
-We use Pre-commit for running lints, tests and other checks locally on commit and push.
-To use pre-commit on your own machine just run:
+We use pre-commit for running lints, tests and other checks locally on commit and push. To use pre-commit on your own machine, you should run the following in your terminal:
 
     sudo pip install pre-commit
     pre-commit install --hook-type pre-commit
     pre-commit install --hook-type commit-msg
     pre-commit install --hook-type pre-push
 
-If you create a pull request your contributed code will automatically be linted, formatted and tested by the GitHub Actions we have set up.
+If you create a pull request, your contributed code will automatically be linted, formatted and tested by the GitHub Action we have set up. New code should always use the same linters and formatters the gliff.ai reposorities currently use. We would also really appreciate new code to come with new tests, where appropriate, to be run against all existing tests!
+
+## Commit Best Practices and Naming Conventions
+
+[{{back to navigation}}](#table-of-contents)
+
+Our commit best practises encourage creating small and frequent commits with clear log messages while editing the source code. This ensures commits are focused and any changes are easily understood and revertible if something goes wrong.
+
+Commit messages should indicate the type and include a short summary of the commit using the following conventions:
+
+- **feat**: The new feature you're adding to a particular application.
```

Comment:
Hmmm... as a team we tend to use `chore` to include `style` and we also use `build`.